app/src/main/java/com/github/dedis/student20_pop/ui/event/PollEventCreationFragment.java

```diff
@@ -0,0 +1,208 @@
+package com.github.dedis.student20_pop.ui.event;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.KeyEvent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.FragmentManager;
+
+import com.github.dedis.student20_pop.PoPApplication;
+import com.github.dedis.student20_pop.R;
+import com.github.dedis.student20_pop.model.Event;
+import com.github.dedis.student20_pop.model.PollEvent;
+import com.github.dedis.student20_pop.utility.ui.organizer.ChoicesListViewAdapter;
+import com.github.dedis.student20_pop.utility.ui.organizer.OnEventCreatedListener;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+public class PollEventCreationFragment extends AbstractEventCreationFragment {
+
+    public static final String TAG = PollEventCreationFragment.class.getSimpleName();
+    public static final DateFormat TIME_FORMAT = new SimpleDateFormat("HH:mm", Locale.FRENCH);
+    public static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy", Locale.FRENCH);
```

Comment:
Duplicate fields

They are already in superclass

Reply:
Right, we can also remove them from MeetingEventCreationFragment